pkg/ebpf/c/tracee.bpf.c

```diff
@@ -3956,12 +3976,22 @@ typedef struct net_event_context {
     net_event_contextmd_t md;
 } net_event_context_t;
 
-struct {
+typedef struct {
+    u64 ts;
+    u16 ip_csum;
+    struct in6_addr ip_saddr;
+    struct in6_addr ip_daddr;
+} indexer_t;
+
+typedef struct {
     __uint(type, BPF_MAP_TYPE_LRU_HASH);
-    __uint(max_entries, 1024);          // simultaneous cgroup/skb ingress/eggress progs
-    __type(key, u64);                   // sk_buff timestamp
+    __uint(max_entries, 10240);         // simultaneous cgroup/skb ingress/eggress progs
```

Comment:
I'll measure this before merging. Need to make sure number here is good for production environments with tons of sockets and on going flows/connections.